index.html

```diff
@@ -223,7 +223,7 @@ <h4>Base rate fallacy</h4>
   </article>
 
   <footer>
-    <a href="/media/esp/puente/codiguitos/uncanny-paginita/contact/index.html">Contact</a>
+    <a href="contact/index.html">Contact</a>
   </footer>
 
 
```

Comment:
I wouldn't leave these empty lines between container and content.

Reply:
Done!